torch/csrc/distributed/c10d/init.cpp

```diff
@@ -2342,12 +2342,19 @@ Example::
       .def_readwrite(
           "is_high_priority_stream",
           &::c10d::ProcessGroupNCCL::Options::is_high_priority_stream)
-      .def_readwrite("config", &::c10d::ProcessGroupNCCL::Options::config);
+      .def_readwrite("config", &::c10d::ProcessGroupNCCL::Options::config)
 #else
       .def_readwrite(
           "is_high_priority_stream",
-          &::c10d::ProcessGroupNCCL::Options::is_high_priority_stream);
+          &::c10d::ProcessGroupNCCL::Options::is_high_priority_stream)
+#endif
```

Comment:
nit: shall we move `is_high_priority_stream` out of #? Since it would exist for both cases.